bin/run-artifactory.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env sh
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" > /dev/null && pwd )"
+
+set -euf
+
+docker run -i -t -d --rm -v "${SCRIPT_DIR}/artifactory.lic:/artifactory_extra_conf/artifactory.lic:ro" \
+  -p 8081:8081 -p 8082:8082  releases-docker.jfrog.io/jfrog/artifactory-pro:7.24.3
+
+echo "Waiting for Artifactory to start"
+until curl -sf -u admin:password http://localhost:8081/artifactory/api/system/licenses/; do
```

Comment:
Is this an example and end users need to replace `admin:password` with their own? Or should this use the env vars mentioned in the README?

Reply:
Actually, this script is strictly just to stand up a local container and test it's readiness. The defaults are used. If it gets changes afterward that's outside the scope of this script.

Users should use env vars when running their TF scripts - but that's covered elsewhere